src/submission/schema/schema-manager.ts

```diff
@@ -356,6 +365,85 @@ namespace MigrationManager {
     return closedMigration;
   };
 
+  const verifyNewSchemaIsValidWithDataValidation = async (
+    newSchemaDictionary: SchemasDictionary,
+  ): Promise<NewSchemaVerificationResult> => {
+    const verificationResult: NewSchemaVerificationResult = {};
+
+    Object.values(ClinicalEntitySchemaNames).forEach(clinicalEntityName => {
+      const clinicalEntityNewSchemaDef = newSchemaDictionary.schemas.find(
```

Comment:
what happens below if this is missing a whole schema, like donor

Reply:
if the a whole schema is missing then its schemaDefinition will be undefined and later on instead of getting all the fields in the schema it will be an empty array. so `checkClinicalEntityNewSchemaHasRequiredFields` will return all the fields in the enum we need for the missing entity in schema.

Reply:
same with missing field code list values